hlint/hlint.dhall

```diff
@@ -76,6 +76,10 @@ in [ rule.Arguments { arguments =
    , hintNote "fmap or (mapM f s)" "anyM f s" "Applying this hint would mean that some actions that were being executed previously would no longer be executed."
    , hintNote "or <$> mapM f s" "anyM f s" "Applying this hint would mean that some actions that were being executed previously would no longer be executed."
 
+   , warnSimple "getAlt . foldMap (Alt . f)" "asumMap"
```

Comment:
I think here it should be `"getAlt (foldMap (Alt . f) xs" "asumMap xs"`

As I recall correctly, `hlint` doesn't handle function composition properly... So by writing `f . g` it won't catch `f (g x)` cases

Reply:
Didn't know about that, thanks

Reply:
I guess there should be both rules.. The same issues applied vice versa